src/interfaces.ts

```diff
@@ -2,13 +2,6 @@ export type Files = Map<string, string>;
 
 export type FileTransform = (files: Files) => Promise<Files>;
 
-export enum VersionState {
-  ready = 'ready',
-  downloading = 'downloading',
-  unzipping = 'unzipping',
-  unknown = 'unknown',
-}
```

Comment:
Why remove the enum here? Is there a way we can retain this type, so we can make sure that only allowed strings are used when working with versions?

Reply:
Oh yeah! But rather than creating a new enum here, it's better if we use the [one defined](https://github.com/electron/fiddle-core/blob/9007a90720c98e46e749b241cfe3d6bbf4d374c3/src/installer.ts#L22) in fiddle-core itself, right ?

Reply:
Oh yeah! Probably, the same should be done in fiddle-core as well right ?
[(one such example)](https://github.com/electron/fiddle-core/blob/9007a90720c98e46e749b241cfe3d6bbf4d374c3/src/installer.ts#L180)